src/serac/physics/utilities/variational_form/tests/l2_comparison.cpp

```diff
@@ -1,149 +0,0 @@
-#include "mfem.hpp"
```

Comment:
Jonathan and I were recently talking about trying to add / debug L2 support, so deleting this might be a little premature. I suppose we can just grab it from an older commit though.

Reply:
That's fair! I think I was being overly ambitious here.